contrib/eos_portable_archive/eos/portable_archive_exception.hpp

```diff
@@ -35,12 +35,11 @@ namespace eos {
 
 	// version of the linked boost archive library
 	const archive_version_type archive_version(
-                11
-//	#if BOOST_VERSION < 103700
-//		boost::archive::ARCHIVE_VERSION()
-//	#else
-//		boost::archive::BOOST_ARCHIVE_VERSION()
-//	#endif
+	#if BOOST_VERSION < 103700
+		boost::archive::ARCHIVE_VERSION()
+	#else
+		boost::archive::BOOST_ARCHIVE_VERSION()
+	#endif
```

Comment:
I'm unsure if this block was previously commented out for MSVC reasons or for any other unknown reason. If that's the case, please advise.